be/src/column/object_column.cpp

```diff
@@ -260,7 +260,6 @@ void ObjectColumn<T>::crc32_hash(uint32_t* hash, uint32_t from, uint32_t to) con
 
 template <typename T>
 int64_t ObjectColumn<T>::xor_checksum(uint32_t from, uint32_t to) const {
-    DCHECK(false) << "object column shouldn't call xor_checksum";
```

Comment:
This change will produce unnecessary scan, for bitmap and other columns ?

Reply:
Better use schema to check.

Reply:
done